tests/validation/tests/v3_api/test_certificate.py

```diff
@@ -0,0 +1,571 @@
+        tls = {"certificateId": self.certificate_all_ns_ssc.id,
+               "hosts": [host_1]}
+        tls_2 = {"certificateId": self.certificate_all_ns_ssc.id,
+                 "hosts": [host_2]}
+        self.ingress = self.p_client.create_ingress(
+            name=ingress_name, namespaceId=self.ns.id, rules=[rule_1],
+            tls=[tls]
+        )
+        wait_for_ingress_to_active(self.p_client, self.ingress)
+        self.p_client.update(
+            self.certificate_all_ns_ssc, key=rancher_private_key,
+            certs=rancher_cert
+        )
+        self.p_client.reload(self.certificate_all_ns_ssc)
+        self.p_client.update(self.ingress, rules=[rule_2], tls=[tls_2])
+        self.p_client.reload(self.ingress)
+        wait_for_ingress_to_active(self.p_client, self.ingress)
+        validate_ingress_using_endpoint(
+            self.p_client, self.ingress, [self.workload], certcheck=True,
+            is_insecure=True
+        )
+
+    @if_test_rbac
+    @pytest.mark.parametrize("role", rbac_role_list)
+    def test_create_certificate(self, role):
+        """
+        Test steps:
+        1. Create certificate for single and all namespace for all role
+        2. Delete the certificates
+        """
+        token = rbac_get_user_token_by_role(role)
+        project = rbac_get_project()
+        ns = rbac_get_namespace()
+        p_client = get_project_client_for_token(project, token)
+        cert_name_1 = random_test_name("cert-rbac")
+        cert_name_2 = random_test_name("cert-rbac")
+        if role in (CLUSTER_MEMBER, PROJECT_READ_ONLY):
+            with pytest.raises(ApiError) as e:
+                p_client.create_namespaced_certificate(
+                    name=cert_name_1, key=rancher_private_key,
+                    certs=rancher_cert,
+                    namespaceId=ns['name']
+                )
+                p_client.create_certificate(
+                    name=cert_name_2, key=rancher_private_key,
+                    certs=rancher_cert
+                )
+                assert e.value.error.status == 403
+                assert e.value.error.code == 'Forbidden'
+        else:
+            certificate_valid = p_client.create_namespaced_certificate(
+                name=cert_name_1, key=rancher_private_key, certs=rancher_cert,
+                namespaceId=ns['name']
+            )
+            assert certificate_valid.issuer == 'Let\'s Encrypt Authority X3'
+            certificate_allns_valid = p_client.create_certificate(
+                name=cert_name_2, key=rancher_private_key,
+                certs=rancher_cert
+            )
+            assert certificate_allns_valid.issuer == 'Let\'s Encrypt ' \
+                                                     'Authority X3'
+            # Delete the resources
+            p_client.delete(certificate_valid)
+            p_client.delete(certificate_allns_valid)
+
+    @if_test_rbac
+    @pytest.mark.parametrize("role", rbac_role_list)
```

Comment:
Can you add a test case that the project owner/member cannot list the namespaces certificates that are not in their project?